websites/P/Pandora/presence.ts

```diff
@@ -1,85 +1,89 @@
 const presence = new Presence({
-    clientId: "608109837657702566"
-  }),
-  strings = presence.getStrings({
-    play: "presence.playback.playing",
-    pause: "presence.playback.paused"
-  });
+  clientId: "608109837657702566"
+}),
+strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused"
+});
 
 function stripText(element: HTMLElement, id = "None", log = true) {
   if (element && element.firstChild) return element.firstChild.textContent;
   else {
     if (log) {
-      presence.info(
-        `%cPandora%cERROR%c An error occurred while stripping data off the page. Please contact Alanexei on the PreMiD Discord server, and send him a screenshot of this error. ID: ${id}`
+      presence.error(
+        `An error occurred while stripping data off the page. Please contact Alanexei on the PreMiD Discord server, and send him a screenshot of this error. ID: ${id}`
       );
     }
     return null;
   }
 }
 
-let state;
-
 presence.on("UpdateData", async () => {
-  let title: HTMLElement,
-    artist: HTMLElement,
-    smallImageKey,
-    smallImageText,
-    audioTime,
-    audioDuration,
-    details,
-    status,
-    audioElement: HTMLAudioElement = document.querySelector("audio:last-child");
-  audioElement === null
-    ? (audioElement = document.querySelector("audio"))
-    : null;
-
-  const audioBar: HTMLElement = document.querySelector(
-    ".Tuner__Audio__NowPlayingHitArea"
-  );
-
-  audioElement && audioBar ? (state = "music") : (state = null);
-
-  switch (state) {
-    case "music":
-      title = document.querySelector(".Tuner__Audio__TrackDetail__title");
-      artist = document.querySelector(".Tuner__Audio__TrackDetail__artist");
-
-      if (title === null && artist === null) return;
-      else {
-        details = stripText(title, "Title");
-        status = stripText(artist, "Title");
-      }
-
-      smallImageKey = "play";
-      smallImageText = (await strings).play;
-      [audioTime, audioDuration] = presence.getTimestamps(
-        Math.floor(audioElement.currentTime),
-        Math.floor(audioElement.duration)
-      );
-      break;
+  // Define presence data
+  const data: PresenceData = {};
 
-    default:
-      details = "Browsing...";
-      break;
-  }
+  // Set default data
+  data.details = "Browsing...";
+  data.largeImageKey = "pandora";
+
+  // Define whether or not we're currently playing
+  const isPlaying = true,
```

Comment:
You have another constant called the same on L65?

Reply:
Whoops, meant to set the variable not define a new one